Modules/Microsoft365DSC/Modules/M365DSCUtil.psm1

```diff
@@ -3174,7 +3174,11 @@ function Get-M365DSCExportContentForResource
 
         [Parameter()]
         [System.Management.Automation.PSCredential]
-        $Credential
+        $Credential,
+
+        [Parameter()]
+        [System.String]
+        $TenantId
```

Comment:
In my use case I'm using this function internally with ConnectionMode set to ServicePrincipalWithThumbprint, without this fix I cannot continue my development, unless I carry this as a local patch.

Since it's not a mandatory parameter adding it everywhere else in M365Dsc can be done at a later stage, adding it like this to the function itself doesn't make it worse.

Reply:
What about using $Results.TenantId instead? Let me test.

Reply:
Yep just tested and it does work, just pushed a new less intrusive diff.